src/logger/loggerApi.c

```diff
@@ -326,7 +351,9 @@ int api_getStatus(struct Serial *serial, const jsmntok_t *json)
         json_int(serial, "trackId", lapstats_get_selected_track_id(), 1);
         json_int(serial, "inLap", (int)lapstats_lap_in_progress(), 1);
         json_int(serial, "armed", lc_is_armed(), 0);
-        json_objEnd(serial, 0);
+        json_objEnd(serial, true);
+
+        get_wifi_status(serial, false);
```

Comment:
I'm ok with that. recommend filing an issue to track refactoring the other functions so the naming is consistent throughout.